app/src/main/java/an/maguste/android/navier/api/RetrofitService.kt

```diff
@@ -0,0 +1,29 @@
+package an.maguste.android.navier.api
+
+import an.maguste.android.navier.BuildConfig
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+
+object RetrofitService {
```

Comment:
Это - холдер. К тому же, слово сервис ассоциируется с Андроид сервисами.
```suggestion
object RetrofitHolder {
```